### ISSUE TITLE
Cant use Controll RetroArch forwader

### ISSUE THREAD

**Issue by StiefelHD** [Root-MtX/Nro2Nsp#29]
I have a problem, when i use the RetroArch forwarder, i cant use the controll...
But in RetroArch works.

In 9.0.0, i use the newest stable and beta version.

**Comment by Root-MtX** [comment]
Update your retroarch cores

**Comment by StiefelHD** [comment]
Thanks.
I actually wanted to stay offline, but it works now.

**Comment by jenergy** [comment]
This solution didn' work for me (SX Os - 9.0.0)
I've updated retroarch cores 
1) manually by downloading official retroarch..without success
2) overriding them from here https://buildbot.libretro.com/nightly/nintendo/switch/libnx/latest/) 
3) directly within the app on the switch.
But nothing changed. When I launch it from Home, keys still don't work.
When I launch retroarch normally and load the same game, it works.
Maybe have I to re-create nsp forwarders by scratch with latest keys?
Or fbalpha and dosbox are not ready yet?

**Comment by StiefelHD** [comment]
Yes i had the same problem, you musst update the core on switch, not on pc manually this dosent work..

**Comment by jenergy** [comment]
I just deleted retroarch on switch sd, and installed the newest version 1.8.0 in a new "retroarch" directory.
I downloaded core fbalpha from the switch retroarch core updater, it was not included in build
I created new nsp forwarder with Nro2Nsp 3.3.6 - Beta 1, using updated keys.dat in resource folder,
using /retroarch/cores/fbalpha_libretro_libnx.nro as path.
Installed new nsp on the switch and..no success. Game loads, but keys does not work.
SXOS latest, fw 9.0.0

I really cannot understand.
